src/components/ImageAndVideoModal.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  Image,
+  Modal,
+  StatusBar,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import Video from 'react-native-video';
+
+export const ImageAndVideoModal = ({visible, setModal, imageOrVideoUri}) => {
+  return (
+    // <StatusBar backgroundColor={'black'}>
```

Comment:
@Charan017 Please remove this comment if unnecessary.

Reply:
Done